activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -15,7 +15,7 @@ def after_teardown # :nodoc:
     end
 
     included do
-      class_attribute :fixture_path, instance_writer: false
+      class_attribute :fixtures_paths, instance_writer: false
```

Comment:
I'm still unsure if this is actually a good idea. We now allow multiple paths but we index fixture cache ivars by fixture set/file names, so 
 `fixture_dir/accounts.yml` and `other_fixture_dir/accounts.yml` doesn't really work. Should I revert the `fixture_dir` deprecation, and maybe just make it a rename to `fixtures_dir`?